docs_in_progress/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/1_cors.md

```diff
@@ -1,73 +1,112 @@
 # CORS
 
-CORS is a [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)
+This service is a [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)
 HTTP-based service inspired by
 [expressjs/cors](https://expressjs.com/en/resources/middleware/cors.html).
+
 Simply add it to your Drash server's `services` config and you are all set!
 
 ## Table of Contents
 
-- [Usage](#usage)
+- [Before You Get Started](#before-you-get-started)
+- [Folder Structure End State](#folder-structure-end-state)
+- [Steps](#configuration)
+- [Verification](#verification)
 - [Configuration](#configuration)
 
-## Usage
+## Before You Get Started
 
-Edit your `deps.ts` file to include `CORS`.
+To use this service, edit your `deps.ts` file to include the service.
 
-```diff
+```typescript
 // deps.ts
 
-export { Drash } from "https://deno.land/drash@v2.0.0/mod.ts";
-+ export { CORS } from "https://deno.land/drash@v2.0.0/services.ts";
+...
+...
+...
+import { CORSService } from "https://deno.land/drash@v2.0.0/services.ts";
+```
+
+## Folder Structure End State
+
+```text
+▾ /path/to/your/project/
+  app.ts
+  deps.ts
 ```
 
+## Steps
+
+1. Create your `app.ts` file.
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
whats going on here lol

Reply:
docs in progress lol.... i was working on getting the steps in, but then started looking into the functionality and couldn't get it to work as i expected